datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -308,10 +337,12 @@ def check_version(self):
 
     @property
     def in_developer_mode(self):
+        # type: () -> bool
         self._log_deprecation('in_developer_mode')
         return False
 
     def get_instance_proxy(self, instance, uri, proxies=None):
+        # type: (InstanceType, str, ProxySettings) -> Any
```

Comment:
Any?

Reply:
Good catch! Should have been `ProxySettings`. Addressed in 3a63e639b.